Gaming The Syllabus/Assets/Scripts/Combat/CombatManager.cs

```diff
@@ -91,6 +99,11 @@ private static void AdvanceCurrentTurnIndex()
         indexOfCombatantWithCurrentTurn = (indexOfCombatantWithCurrentTurn + 1) % allCombatants.Count;
     }
 
+    public static CombatUnit getCurrentCombatant()
```

Comment:
This might make more sense as a property

Reply:
But very good call not letting other classes have public access to the allCombatants list